package.json

```diff
@@ -15,7 +15,8 @@
     "build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
     "lint": "eslint .",
     "dev": "cross-env BABEL_ENV=default webpack --watch",
-    "test": "npm run lint && npm run test-unit"
+    "test": "npm run lint && npm run test-unit",
+	"travis-builds": "concurrently \"npm run build\" \"cross-env BABEL_ENV=default webpack\" \"cross-env BABEL_ENV=default NODE_ENV=test webpack\""
```

Comment:
I don't know that we need to build all the environments. I think this could be simplified to:

```json
"ci": "concurrently \"npm run build\" \"npm test\""
```

(Assuming `script` instead of `before_script` per my previous suggestion)

Aside: How do you feel about the `ci` name suggestion here? I don't really have a strong preference; seemed simpler and less specific to any one CI tool in case we'd ever change.

Can we also fix the whitespace here to use spaces instead of tabs? Spaces are an exception for this file because the default behavior of `npm install --save` (and `--save-dev`) will use spaces anyways. We define an [EditorConfig](http://editorconfig.org/) configuration for the project which enforces this ([source](https://github.com/WordPress/gutenberg/blob/7b388897315a2faf4b911fc88d3179cc7f1e993a/.editorconfig#L11-L13)). Installing a plugin for your editor will avoid the need for you to account for this manually.